_data/apis.json

```diff
@@ -14866,5 +14866,836 @@
             "zh_cn",
             "zh_tw"
         ]
+    },
+    {
+        "name": "OpenAI GPT",
+        "id": "openai-gpt",
+        "company": "openai",
+        "urls": [
+            "https://platform.openai.com/docs/guides/gpt",
+            "https://platform.openai.com/docs/guides/prompt-engineering"
+        ],
+        "languages": [
+            "af",
+            "ar",
+            "az",
+            "be",
+            "bg",
+            "bs",
+            "ca",
+            "cs",
+            "cy",
+            "da",
+            "de",
+            "el",
+            "en",
+            "es",
+            "et",
+            "fa",
+            "fi",
+            "fr",
+            "gl",
+            "he",
+            "hi",
+            "hu",
+            "hy",
+            "id",
+            "is",
+            "it",
+            "ja",
+            "kn",
+            "kk",
+            "ko",
+            "lv",
+            "lt",
+            "mk",
+            "mi",
+            "mr",
+            "ms",
+            "ne",
+            "nl",
+            "no",
+            "pl",
+            "pt",
+            "ro",
+            "ru",
+            "sk",
+            "sl",
+            "sr",
+            "sv",
+            "sw",
+            "ta",
+            "th",
+            "tl",
+            "tr",
+            "uk",
+            "ur",
+            "vi",
+            "zh"
+        ],
+        "text_translation": true,
+        "speech_translation": true,
+        "file_translation": true,
+        "self-serve": true,
+        "gen_ai": true
```

Comment:
there's basically 3 groups:

1. "traditional" APIs
2. GenAI, but translation-specific (TowerLLM, Lara...) i.e. it does not require a prompt
3. GenAI, not translation-specific, so it requires a prompt

so some options:

- `"translation_specific": false`
- `"type": "gen_ai" / "translation_gen_ai" / ...`
- `"prompt_required": true`